src/Dashboard/Web.config

```diff
@@ -1,17 +1,17 @@
-﻿<?xml version="1.0"?>
```

Comment:
Some of the changes, like in this file for example, are things that VS Code changed on it's own. I didn't do this manually and I'm not sure why it made these weird changes anyway. I've gone through the review to revert most of these random changes but I don't know why it did it in the first place. It might have been some things it did when I had build issues.